src/pages/AdminPage/User/DriverInfo/DriverDetail/index.jsx

```diff
@@ -118,7 +142,7 @@ function DriverDetail() {
       bank: driverInfo.bank,
       holidays: driverInfo.holidays,
       introduction: driverInfo.introduction,
-      phoneNumber: driverInfo.phoneNumber,
+      phoneNumber: newPhoneNum,
       prices: driverInfo.prices,
       profileImg: profileImageURL,
       region: driverInfo.region,
```

Comment:
```suggestion
      region: driverInfo.region,
      driverLicenseImg: driverInfo.driverLicenseImg,
      taxiLicenseImg: driverInfo.taxiLicenseImg,
      insuranceLicenseImg: driverInfo.insuranceLicenseImg,
```

자동 저장에는 라이센스 이미지들이 빠져있는 것 같아
그래서 지금은 실제로 자동 저장에서 오류가 발생하고 있어

Reply:
자동 저장은 왜 빼고 작업했는지 모르겠다..ㅜ.. 다음부턴 더 신경써서 작업해볼게!
해당 사항 반영해서 다시 PR 날렸어